LuhnAlgorithm/src/main/java/com/exercism/luhn/Mod10Validator.java

```diff
@@ -0,0 +1,84 @@
+package com.exercism.luhn;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Mod10Validator {
+    private static final String FILE_NAME = "TestData.txt";
+    private ArrayList<String> listToValidate;
+    Scanner scanner;
```

Comment:
you don't need this as a `Mod10Validator` field. Basically, you use the Scanner only to read a file, so the logical approach would be to also put it inside the `fileReader()` method